src/components/Slot/Slot.tsx

```diff
@@ -56,7 +56,16 @@ class Slot extends UIComponent<Extendable<ISlotProps>, any> {
   }
 
   static create = createSlotFactory(Slot.defaultProps.as, content => ({ content }))
+
+  // ============================================================
+  // HTML Factories
+  // ============================================================
+  static createHTMLDivision = createSlotFactory('div', children => ({ children }))
+  static createHTMLIframe = createSlotFactory('iframe', src => ({ src }))
+  static createHTMLImage = createSlotFactory('img', src => ({ src }))
   static createHTMLInput = createSlotFactory('input', type => ({ type }))
+  static createHTMLLabel = createSlotFactory('label', children => ({ children }))
+  static createHTMLParagraph = createSlotFactory('p', children => ({ children }))
```

Comment:
Per last sync, let's not create statics on Slot.  We'll also not have dedicated factories for each HTML element. 
 This is because we can use the `as` prop instead for all these cases.

The proper `mapValueToProps` function will be chosen by the factory according to the final computed element type.  This will be done in the `createShorthand()` method in `factories.tsx`.

Reply:
changes were made @levithomason can u please another look?